src/v2/client.tsx

```diff
@@ -62,6 +63,11 @@ async function setupClient() {
   } catch (error) {
     console.error("[v2/client.tsx] Error loading client:", error)
   }
+  try {
+    await setupBraze()
+  } catch (error) {
+    console.error("Braze marketing sdk setup failed:", error)
```

Comment:
This is silly and minor, but just above we're formatting the logs slightly different and [in a specific way](https://github.com/artsy/force/pull/7363/files#diff-6fc65e0cafc54cf86fbed0faf89d434918d2bd0ce4f11162befcd9155da0f8b8R64) (with file path): 

This prefix path to log has saved us countless times when dealing with minification, server logs, bad sourcemaps, etc, so I always try to encourage it.